pkg/cmd/experimental/syncgroups/syncgroups.go

```diff
@@ -38,19 +38,20 @@ to sync and where their records live. For instance, all or some groups may be se
 stored in OpenShift that have been synced previously, or similarly all or some groups may be selected from those 
 stored on an LDAP server. The path to a sync configuration file is required in order to describe how data is 
 requested from the external record store and migrated to OpenShift records. Default behavior is to sync all 
-groups from the LDAP server returned by the LDAP query templates.
+groups from the LDAP server returned by the LDAP query templates. The sync job defaults to a dry run, and will 
+only commit changes if the --confirm flag is present.
 `
 	syncGroupsExamples = `  // Sync all groups from an LDAP server
-  $ %[1]s --sync-config=/path/to/ldap-sync-config.yaml
+  $ %[1]s --sync-config=/path/to/ldap-sync-config.yaml --confirm
 
   // Sync specific groups specified in a whitelist file with an LDAP server 
-  $ %[1]s --whitelist=/path/to/whitelist.txt --sync-config=/path/to/sync-config.yaml
+  $ %[1]s --whitelist=/path/to/whitelist.txt --sync-config=/path/to/sync-config.yaml --confirm
 
   // Sync all OpenShift Groups that have been synced previously with an LDAP server
-  $ %[1]s --existing --sync-config=/path/to/ldap-sync-config.yaml
+  $ %[1]s --type=openshift --sync-config=/path/to/ldap-sync-config.yaml --confirm
 
   // Sync specific OpenShift Groups if they have been synced previously with an LDAP server
-  $ %[1]s groups/group1 groups/group2 groups/group3 --sync-config=/path/to/sync-config.yaml
+  $ %[1]s groups/group1 groups/group2 groups/group3 --sync-config=/path/to/sync-config.yaml --confirm
```

Comment:
> This needs to specify the type. whitelist-type?

Or any other suggestion?  I dislike type, but I can't really think of anything short and sweet.